src/secure_random.cr

```diff
@@ -10,16 +11,23 @@ require "base64"
 #
 # Examples:
 # ```crystal
-# SecureRandom.base64 # => "LIa9s/zWzJx49m/9zDX+VQ=="
-# SecureRandom.hex    # => "c8353864ff9764a39ef74983ec0d4a38"
-# SecureRandom.uuid   # => "c7ee4add-207f-411a-97b7-0d22788566d6"
+# SecureRandom.random_bytes(8) # => Bytes[141, 161, 130, 39, 247, 150, 68, 233]
+# SecureRandom.base64          # => "LIa9s/zWzJx49m/9zDX+VQ=="
+# SecureRandom.hex             # => "c8353864ff9764a39ef74983ec0d4a38"
+# SecureRandom.uuid            # => "c7ee4add-207f-411a-97b7-0d22788566d6"
+#
+# SecureRandom.rand(10000)        # => 4264
+# [1, 2, 3].shuffle(SecureRandom) # => [1, 3, 2]
 # ```
 #
 # The implementation follows the
 # [libsodium sysrandom](https://github.com/jedisct1/libsodium/blob/6fad3644b53021fb377ca1207fa6e1ac96d0b131/src/libsodium/randombytes/sysrandom/randombytes_sysrandom.c)
 # implementation and uses `getrandom` on Linux (when provided by the kernel),
 # then tries to read from `/dev/urandom`.
 module SecureRandom
+  extend Random
```

Comment:
It's a module which can't be instantiated. I was really surprised that because `extend` is used, the `SecureRandom` **module** itself can be used exactly like an **instance** of a class including `Random`